src/App.css

```
@@ -1,13 +1,61 @@
+:root{
+  --primary-color: #4caf50;
+  --error-color: #F44336;
+  --regular-shadow:rgba(149, 157, 165, 0.2) 0px 8px 24px;
```

```suggestion
  --regular-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
```
won't prettier handle this?